README.md

```diff
@@ -0,0 +1,45 @@
+# Clickbait Filter
+
+This project is an optional challenge of the mandatory course 
+"Machine Learning for Natural Language Understanding" of my NLP master's degree
+program at Trier University in the winter semester 22/23. 
+
+## Task
+
+The Task was to train a clickbait filter to classify clickbait articles 
+by their headline. I could freely decide how to prepare the data and which ML 
+model to use for classification.
+
+The challenge was considered passed if the model performs better than professor's 
+baseline (a simple classifier; F1 ~0.89). 
+
+## Dataset
+The data consists of two files, a text file with clickbait 
+headlines and one with headlines from news sources. 
+The hold out dataset is organized the same way.
+
+I'm not allowed to publish the train and validation datasets since they are a property of
+Computerlinguistik und Digital Humanities Department of the University of Trier.
+
+## Results
+
+I implemented an [LSTM model](https://pytorch.org/docs/stable/generated/torch.nn.LSTM.html) (Raschka, 2022, p.499)
+with [dropouts](https://pytorch.org/docs/stable/generated/torch.nn.Dropout.html) using PyTorch library (`./utils/models.py`)
+It showed a quite good result on the validation set: ___F1-Score = 96.2%___ (`./notebooks/validation_and_examples.ipynb`) which is however can be easily overcome with Transformer architecture.
```

Comment:
I suggest adding a confusion matrix and presition&recall.

Reply:
It is better to either add notebook with experiments with transformers or delete sentence about "Transformer"